src/FileUtil/FileUtil.spec.js

```diff
@@ -48,10 +48,6 @@ describe('FileUtil', () => {
       map = TestUtil.createMap();
     });
 
-    afterEach(() => {
-      TestUtil.removeMap(map);
-    });
```

Comment:
Why is this needed?

Reply:
Map removal leads to the following error, cause [`renderer`](https://github.com/openlayers/openlayers/blob/master/src/ol/PluggableMap.js#L1292) seems to be null at the moment where the test tries to access the map then:
```
TypeError: Cannot read property 'renderFrame' of null

at Map.PluggableMap.renderFrame_ (node_modules/ol/src/PluggableMap.js:1292:20)
      at Map.<anonymous> (node_modules/ol/src/PluggableMap.js:198:12)
      at Timeout.callback [as _onTimeout] (node_modules/jsdom/lib/jsdom/browser/Window.js:678:19)
```

The error is possibly due to https://github.com/openlayers/openlayers/pull/9637

For now, I've commented the `afterEach` test block and  added a `TODO` to investigate it later.

I've also added the corresponding issue.